dotnet/src/Connectors/Connectors.AI.OpenAI/TextEmbedding/AzureTextEmbeddingGeneration.cs

```diff
@@ -12,12 +13,12 @@
 namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.TextEmbedding;
 
 /// <summary>
-/// Azure OpenAI text embedding service.
```

Comment:
I think service was the right term we're going with as `ITextEmbeddingGeneration` implements `IAIService`

Reply:
I see though we were not consistent in this file already. /shrug